wdl-format/src/config/indent.rs

```diff
@@ -1,22 +1,50 @@
 //! Indentation within formatting configuration.
 
-use std::num::NonZeroUsize;
+use crate::SPACE;
+use crate::TAB;
 
 /// The default indentation.
-pub const DEFAULT_INDENT: Indent = Indent::Spaces(unsafe { NonZeroUsize::new_unchecked(4) });
+pub const DEFAULT_INDENT: Indent = Indent::Spaces(4);
+/// The maximum number of spaces to represent one indentation level.
+pub const MAX_SPACE_INDENT: usize = 16;
 
 /// An indentation level.
 #[derive(Clone, Copy, Debug)]
 pub enum Indent {
     /// Tabs.
-    Tabs(NonZeroUsize),
-
+    Tabs,
```

Comment:
So if a user indents with `tab`, they can only use one tab. I'm fine with that, but just flagging that to make sure we're OK with that choice.